brage-import/src/test/java/no/sikt/nva/brage/migration/lambda/BrageEntryEventConsumerTest.java

```diff
@@ -1895,6 +1896,40 @@ void shouldMergeConferenceReportWithEvent() throws IOException {
                    is(equalTo(existingPublication.getIdentifier())));
     }
 
+    @Test
+    void whenSearchApiReturnsMultiplePublicationsButDynamoDbCannotFindTheFirstOneMergingShouldHappenWithNextInLine()
+        throws IOException {
+        // create a brage record and matching publication
+        var publication = randomPublication(ConferencePoster.class);
+        publication.setAdditionalIdentifiers(Set.of());
+        publication.getEntityDescription().getReference().setDoi(null);
+        publication.getEntityDescription().setPublicationDate(new no.unit.nva.model.PublicationDate.Builder().withYear("2022").build());
+        publication.getEntityDescription().setMainTitle("Dynamic - Response of Floating Wind Turbines! Report");
+        var existingPublication = resourceService.createPublicationFromImportedEntry(publication);
+        var instanceType = existingPublication.getEntityDescription().getReference().getPublicationInstance().getInstanceType();
+        var contributor = existingPublication.getEntityDescription().getContributors().getFirst();
+        var brageContributor = new Contributor(new Identity(contributor.getIdentity().getName(), null),
+                                               "ARTIST", null, List.of());
+        var generator = new NvaBrageMigrationDataGenerator.Builder()
+                            .withMainTitle("Dynamic Response of Floating Wind Turbines")
+                            .withContributor(brageContributor)
+                            .withPublicationDate(new PublicationDate("2023",
+                                                                     new PublicationDateNva.Builder().withYear("2023").build()))
+                            .withType(new Type(List.of(), instanceType)).build();
```

Comment:
We have a method for persisting duplicate

`var generator = generateBrageRecordAndPersistDuplicate(new Lecture(), TYPE_CONFERENCE_REPORT);`

Reply:
nice will change.